sdk/core/azure-json-reflect/src/main/java/com/azure/json/reflect/JsonFactoryBuilder.java

```diff
@@ -0,0 +1,22 @@
+package com.azure.json.reflect;
+
+import com.azure.json.reflect.gson.GsonJsonFactory;
+import com.azure.json.reflect.jackson.JacksonJsonFactory;
+
+public class JsonFactoryBuilder {
```

Comment:
This isn't really a 'builder' - it probably better suits the name `JsonFactory` that is already used above, with the method named `getInstance()`. But I also wonder if this method should have a no-args private constructor (so it can't be instantiated) and the build/getInstance method be static, so that people can call `JsonFactory.getInstance()`?